### DISCUSSION TITLE
Entities Not available

### DISCUSSION THREAD

**Discussion by urkman5000** [tsightler/ring-mqtt#697] [General Discussion]
Hey there, ATM the entities of the Ring Intercom are all anavailable. 
Dunno when this began. Already re-auth the token but no Change at all.

The Post comes daily and also the Calle of the Delivery man havnt gotten through. In the Last couple days.

![IMG_0722](https://github.com/tsightler/ring-mqtt/assets/52527379/f1ab61c1-3bcd-452a-8608-1467e92cd9b8)

**Comment by magnetic157** [comment]
I'm having the same issue, all of my entities became unavailable and I can't get them back up.  The UI shows a successful connection with the API Key.  It's been this way for an over a month now.

**Comment by vincentfhylla** [comment]
Same Issue

**Comment by HQOFIQ** [comment]
Had the same issue - I just restarted Ring-MQTT in Home Assistant (think the update of HASS was the breaking moment), runs smooth again, everything available

**Reply by tub5** [reply]
Just tried that to no avail. Rather annoying thing to wake up to.

**Reply by AgNasty** [reply]
Same issue here. Nothing in the logs, connection comes up healthy. Refreshed ring connection as well. Ensured everything was up to date.

**Reply by tub5** [reply]
@AgNasty managed to get it back by restarting the base station via the Ring app, then Ring MQTT addon, then MQTT itself

**Reply by HQOFIQ** [reply]
I suggest you use something like MQTT Explorer (Desktop App) and connect to the MQTT-Stream to see what is broadcasted - in my case only the cams were in there and somehow the sensors, keypads, etc. were missing, restarting the MQTT did the trick and everything was broadcasted again

**Reply by AgNasty** [reply]
I'm in that exact scenario. I see the two cameras but none of the other devices that are attatched to the base station. Which thinking about it makes sense that it is a base station issue since the cameras are independent devices in Ring. Trying the reboot again with pihole disable. I saw that Ring was trying to perform an update but I didn't see if it was successful. 

Okay, I can see everything in the mqtt-explorer and HASS. So it was either the second reboot of Ring base station or the update that was finally completed with unrestricted web traffic. All is well again. As silly as it sounds the automatic door locking automation I made was the only thing my wife cared about in HASS so this was a top priority haha. 

Thanks for the suggestions @tub5 and @HQOFIQ
